models/public/ssd_resnet50_v1_fpn_coco/model.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The "ssd_resnet50_v1_fpn_coco" model is a SSD FPN object detection architecture based on Resnet50 (https://arxiv.org/abs/1708.02002).
+  The model has been trained from the Common Objects in Context (COCO) image dataset.
+  Model repository: https://github.com/tensorflow/models/blob/master/research/object_detection
+  Origanal model can be downloaded from here: http://download.tensorflow.org/models/object_detection/ssd_resnet50_v1_fpn_shared_box_predictor_640x640_coco14_sync_2018_07_03.tar.gz
```

Comment:
not sure that link to model should be placed in description (otherwise, why we need model downloader if we just provide the link :))

Reply:
Link has been deleted